consent-ws/src/main/java/org/broadinstitute/consent/http/service/DatabaseDataAccessRequestAPI.java

```diff
@@ -116,15 +116,15 @@ public void setMongoDBInstance(MongoConsentDB mongo) {
             dataAccessRequest.remove(DarConstants.ID);
             dataAccessRequest.remove(DarConstants.PARTIAL_DAR_CODE);
         }
-        List<String> dataSets =  dataAccessRequest.get(DATA_SET_ID, List.class);
-        dataAccessRequest.remove(DATA_SET_ID);
+        List<Integer> dataSets =  dataAccessRequest.get(DATA_SET_ID, List.class);
         if (CollectionUtils.isNotEmpty(dataSets)) {
-            Set<ConsentDataSet> consentDataSets = consentDAO.getConsentIdAndDataSets(dataSets);
+            Set<ConsentDataSet> consentDataSets = consentDAO.getConsentIdAndDataSets(dataAccessRequest.get(DATA_SET_ID, List.class));
```

Comment:
why are you making the get request again instead of just using the dataSets variable like before?